.github/labeler.yml

```diff
@@ -1,6 +1,6 @@
 # This configures the .github/workflows/pull_request_label.yml workflow. 
 
-'infra':
+'type-infra':
```

Comment:
Uh. What does this do? I have no context. 🤷

Reply:
This is just changing the label we apply to for infra related PRs - from `infra` to `type-infra`. It's not a fully formed thought yet, but I'll likely push for kebob-style labels for dart-lang repos (ala https://github.com/dart-lang/sdk/labels).